### ISSUE TITLE
NameError: name 'dev' is not defined

### ISSUE THREAD

**Issue by mirsella** [robinuniverse/Keebie#1]
Hello, i'm trying to get Keebie working.
here the error i got :
```
[mirsella@predator ~/Documents/Keebie]$ python keebie.py --add
Welcome to Keebie
Enter the command you would like to attribute to a key on your second keyboard 
echo ergkljh
Please press the key you would like to assign the command to...
Traceback (most recent call last):
  File "keebie.py", line 132, in <module>
    addKey()
  File "keebie.py", line 59, in addKey
    for event in dev.read_loop():
NameError: name 'dev' is not defined
```
seems like dev is not set in the addKey() func.
after some documentation and reading your code (i'm a newbie, so i'm happy for what a just done) and i just added `dev = InputDevice(config()[0])` at the start of the function addKey(). God i love open source, being able to do that. now it's working great, this script is awesome ! 
tell me if you add a better solution to solve this issue thanks

**Comment by craftycodie** [comment]
Hi @mirsella!
Having had a look, you're right.

The only issue with this solution is that dev is not always assigned InputDevice(config()[0])
As you can see on line 134, it may also be assigned InputDevice("/dev/input/by-id/"+args.device), based on a given argument.

In any case, I've made a Pull Request that should resolve this issue (#2 ).

Thanks for the info!

Edit: Oh nvm that was not intended functionality.

**Comment by daisyUniverse** [comment]
NICE NICE NICE NICE NICE NICE YOU FIXED IT

**Comment by daisyUniverse** [comment]
Fixed #2
